os/include/debug.h

```diff
@@ -1146,6 +1146,30 @@ int get_errno(void);
 #define videoivdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_XX_ERROR
```

Comment:
Hi @namanjain7 , I think you misunderstood @sunghan-chang explanation, he is referring that when we add a new driver module, we will add log printing according to the module naming. For example, LCD, we use lcddbg, lcdvdbg.
Thus for MIPI, you should add like mipidbg, mipivdbg, etc.

Reply:
done